pulseapi/utility/tests/test_newsletter.py

```diff
@@ -1,44 +0,0 @@
-import json
```

Comment:
change history of this file: https://github.com/mozilla/network-pulse-api/commits/master/pulseapi/utility/tests/test_newsletter.py